content/ui-patterns/saving.mdx

```diff
@@ -241,27 +304,45 @@ If the change is not obvious within the user's viewport, provide feedback to let
 - If the change occurred without a redirect or page refresh, you can use the [toast component](https://primer.style/design/ui-patterns/messaging#toasts).
 - If the change occurred and the page refreshes or redirects, you can use a [flash banner component](https://primer.style/design/ui-patterns/messaging#flash-alerts).
 
-<Note variant="warning">If your design is being implemented in github.com, avoid using a toast for now. The toast components used on github.com have accessibility issues that need to be fixed.</Note>
+<Note variant="warning">
+  If your design is being implemented in github.com, avoid using a toast for now. The toast components used on
+  github.com have accessibility issues that need to be fixed.
+</Note>
 
 ## Error forgiveness
 
 Before submitting a form with potentially destructive consequences (for example, delete a repository), ask the user to confirm they want to submit the form.
 
 For risky changes, provide a way to undo those changes after saving.
 
-<Box display="flex" alignItems="center" flexDirection="column"mx="auto" sx={{gap: 1}}>
-    <Box display="inline-block" overflow="hidden" borderWidth="1px" borderColor="border.default" borderStyle="solid" borderRadius="12px">
-        <img width="960" src="https://user-images.githubusercontent.com/2313998/152434199-a067b037-baf8-4093-8796-b0bb58344223.png" />
-    </Box>
-    <Text as="p" mt="0" align="center" fontSize={1}>After merging a pull request, a button appears to revert the changes</Text>
+<Box display="flex" alignItems="center" flexDirection="column" mx="auto" sx={{gap: 1}}>
+  <Box
+    display="inline-block"
+    overflow="hidden"
+    borderWidth="1px"
+    borderColor="border.default"
+    borderStyle="solid"
+    borderRadius="12px"
+  >
+    <img
+      width="960"
```

Comment:
Missing alt tag of presentation role, screen reader will attempt to read the URL.